### PR TITLE
Editor Preview

### PR DESCRIPTION
**Pull request by brianp**
### Summary

Make the editor preview a side by side preview.

> I assign all rights, including copyright, to any future Dradis
> work by myself to Security Roots.

### Check List

- [x] Added a CHANGELOG entry

### CODE REVIEW

File: vendor/assets/javascripts/jquery.textile.js [hunk 1]
```diff
@@ -69,14 +69,28 @@
...
+      // add Preview to container and load
       this.options.$preview = $(this.options.tpl.preview);
       $('.textile-inner', this.options.$wrap).append(this.options.$preview);
-      this.options.$preview.hide();
+      this.options.$preview.wrap('<div class="col-6"></div>');
+      this._loadPreview();
+
+      // Sync preview
+      var typingTimer;
+      var doneTypingInterval = 800;
```
Comment (aapomm) [inline vendor/assets/javascripts/jquery.textile.js]: This same feature in the plugin manager uses 500ms as a buffer. It might be good to be consistent.

Comment (brianp) [reply vendor/assets/javascripts/jquery.textile.js]: I'll update it for consistency. I couldn't find it in the plugin manager though, have a link?

Comment (brianp) [reply vendor/assets/javascripts/jquery.textile.js]: @aapomm pinged this on slack: https://github.com/secroots/dradis-pro/blob/develop/app/assets/javascripts/mintcreek/modules/admin/plugins.js.coffee#L12

I'm going to update it to match the `textchange` we use, and remove the onkeydown event I added just for consistency.

File: CHANGELOG [hunk 2]
```diff
@@ -8,6 +8,7 @@ v3.17 (Month 2020)
   * Refactored dots-menu to make it available in any view
   * Remove tag color from issue titles in issue summary
   * Use shared noscript partial
+  * Make the editor preview a side-by-side preview that auto-updates
   * Upgraded gems: puma, sass-rails
```
Comment (etdsoft) [inline CHANGELOG:12]:
```suggestion
  * Remove tag color from issue titles in issue summary
  * Side-by-side editor preview that auto-updates
  * Use shared noscript partial
  * Upgraded gems: puma, sass-rails
```

File: CHANGELOG [hunk 3]
```diff
@@ -16,6 +17,7 @@ v3.17 (Month 2020)
     - Sidebar items not showing active state
     - Stop bin/setup from creating folders outside dradis-ce/
     - bin/setup will not error if the attachments directory already exists
+    - Fix textile preview not showing on issues with very long text
```
Comment (etdsoft) [inline CHANGELOG:20]: Can you please address the :alpha

Comment (brianp) [reply CHANGELOG:20]: CHANGELOG in alpha? Now I've seen it all 🙈

Will do.

**Pull request merged**